web/impact/requirements/dev.txt

```diff
@@ -11,3 +11,4 @@ git+https://github.com/PyCQA/astroid.git#egg=astroid
 git+https://github.com/PyCQA/pylint#egg=pylint
 ipdb==0.8.1
 ipython==4.0.0
+jsonschema
```

Comment:
do we want to pin it?

Reply:
No.  I want a system that pins it for me.  See other tickets about that.